hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/SubProcessHapiTestNode.java

```diff
@@ -332,6 +330,7 @@ private String getClasspath() {
                 .collect(Collectors.joining(":"));
     }
 
+    @SuppressWarnings("java:S2142")
```

Comment:
Can you please add a comment on which warning we are suppressing here?

Reply:
done sure